pyresample/test/test_geometry.py

```diff
@@ -701,6 +597,179 @@ def test_get_xy_from_lonlat(self):
+        expected = np.array([1, 2, 3, 4, 5, 6])
+        self.assertTrue(np.array_equal(swath_def1.lons, expected) and
+                        np.array_equal(swath_def1.lons, expected),
+                        'Failed to append 1D swaths')
+
+    def test_append_2d(self):
+        lons1 = np.array([[1, 2, 3], [3, 4, 5], [5, 6, 7]])
+        lats1 = np.array([[1, 2, 3], [3, 4, 5], [5, 6, 7]])
+        lons2 = np.array([[4, 5, 6], [6, 7, 8]])
+        lats2 = np.array([[4, 5, 6], [6, 7, 8]])
+        swath_def1 = geometry.SwathDefinition(lons1, lats1)
+        swath_def2 = geometry.SwathDefinition(lons2, lats2)
+        swath_def1.append(swath_def2)
+        expected = np.array(
+            [[1, 2, 3], [3, 4, 5], [5, 6, 7], [4, 5, 6], [6, 7, 8]])
+        self.assertTrue(np.array_equal(swath_def1.lons, expected) and
+                        np.array_equal(swath_def1.lons, expected),
+                        'Failed to append 2D swaths')
+
+    def test_swath_equal(self):
+        """Test swath equality."""
+        lons = np.array([1.2, 1.3, 1.4, 1.5])
+        lats = np.array([65.9, 65.86, 65.82, 65.78])
+        swath_def = geometry.SwathDefinition(lons, lats)
+        swath_def2 = geometry.SwathDefinition(lons, lats)
+        self.assertFalse(
+            swath_def != swath_def2, 'swath_defs are not equal as expected')
+
+    def test_swath_not_equal(self):
+        """Test swath inequality."""
+        lats1 = np.array([65.9, 65.86, 65.82, 65.78])
+        lons = np.array([1.2, 1.3, 1.4, 1.5])
+        lats2 = np.array([65.91, 65.85, 65.80, 65.75])
+        swath_def = geometry.SwathDefinition(lons, lats1)
+        swath_def2 = geometry.SwathDefinition(lons, lats2)
+        self.assertFalse(
+            swath_def == swath_def2, 'swath_defs are not expected to be equal')
+
+    def test_compute_omerc_params(self):
+        """Test omerc parameters computation."""
+        lats = np.array([[85.23900604248047, 62.256004333496094, 35.58000183105469],
+                         [80.84000396728516, 60.74200439453125, 34.08500289916992],
+                         [67.07600402832031, 54.147003173828125, 30.547000885009766]]).T
+
+        lons = np.array([[-90.67900085449219, -21.565000534057617, -21.525001525878906],
+                         [79.11000061035156, 7.284000396728516, -5.107000350952148],
+                         [81.26400756835938, 29.672000885009766, 10.260000228881836]]).T
+
+        area = geometry.SwathDefinition(lons, lats)
+        proj_dict = {'no_rot': True, 'lonc': 5.340645620216994,
+                     'ellps': 'WGS84', 'proj': 'omerc',
+                     'alpha': 19.022450179020247, 'lat_0': 60.7420043944989}
+        self.assertDictEqual(area._compute_omerc_parameters('WGS84'),
+                             proj_dict)
+
+    def test_get_edge_lonlats(self):
+        """Test the `get_edge_lonlats` functionality."""
+        lats = np.array([[85.23900604248047, 62.256004333496094, 35.58000183105469],
```

Comment:
Do you test an area containing the North/South pole?

Reply:
Not yet :)